robottelo/api/utils.py

```diff
@@ -891,3 +891,27 @@ def attach_custom_product_subscription(prod_name=None, host_name=None):
         query={'search': 'name={0}'.format(prod_name)})[0]
     entities.HostSubscription(host=host.id).add_subscriptions(
         data={'subscriptions': [{'id': product_subscription.id, 'quantity': 1}]})
+
+
+def update_provisioning_template(name=None, old=None, new=None, org_id=None):
+    """ Update provisioning template content
+
+    :param str name: template provisioning name
+    :param str old: current content
+    :param str new: replace content
+    :param int org_id: organization ID
+
+    :return boolean: True/False
+    """
+    temp = entities.ProvisioningTemplate().search(
+        query={
+            'per_page': 1000,
+            'search': 'name="{}"'.format(name),
+            'organization_id': org_id
+        })[0].read()
+    if old in temp:
+        temp.template = temp.template.replace(old, new)
+        update = temp.update(['template'])
+        return new in update.template
+    else:
+        return False
```

Comment:
Better if we can raise an error here saying the old isnt available. Else the user will confuse why its failed wheather due to New in update or old in temp.

Reply:
Added :)